module-services/service-appmgr/model/ApplicationManagerCommon.cpp

```diff
@@ -157,14 +157,13 @@ namespace app::manager
         ActionRequest act = ActionRequest{this->GetName(), app::manager::actions::DisplayLogoAtExit, nullptr};
         switch (closeReason) {
         case sys::CloseReason::SystemBrownout:
-            [[fallthrough]];
         case sys::CloseReason::LowBattery:
             act = ActionRequest{this->GetName(), app::manager::actions::SystemBrownout, nullptr};
             break;
         case sys::CloseReason::RegularPowerDown:
-            [[fallthrough]];
         case sys::CloseReason::Reboot:
-            [[fallthrough]];
```

Comment:
why did you remove the [[fallthrough]] modifiers?

Reply:
Because they seem to be used incorrectly.
The compiler may warn about unintentional fallthrough, if a `case` is ended without `break` or `[[fallthrough]]` and have at least one statement. With an empty `case` the compiler will not warn.

Reply:
sure, isocpp says the same ;)
https://isocpp.org/files/papers/P0188R0.pdf